lmod_proxy/edx_grades/actions.py

```diff
@@ -30,6 +30,16 @@ def post_grades(gradebook, form):
     csv_file.seek(0)
     results = None
     try:
+        # Create assignment to explicitly set max points.
+        assignment, max_points = _get_assignment_name_max_points(form)
+        gradebook.create_assignment(
+            assignment[0:3] + assignment[-2:],
```

Comment:
This looks like it was adapted from https://github.com/mitodl/PyLmod/blob/master/pylmod/gradebook.py#L916